virttest/qemu_vm.py

```diff
@@ -1439,13 +1439,15 @@ def add_sga(devices):
         def add_watchdog(devices, device_type=None, action="reset"):
             watchdog_devs = []
             parent_bus = None
-            if device_type and devices.has_device(device_type):
+            if device_type == "itco":
+                dev = qdevices.QGlobal('ICH9-LPC', 'noreboot', 'off')
+            elif device_type and devices.has_device(device_type):
                 if devices.is_pci_device(device_type):
                     parent_bus = self._get_pci_bus(self.params, None, False)
                 dev = QDevice(device_type, parent_bus=parent_bus)
                 set_cmdline_format_by_cfg(dev, self._get_cmdline_format_cfg(),
                                           "watchdog")
-                watchdog_devs.append(dev)
+            watchdog_devs.append(dev)
```

Comment:
Here the issue is that a `NameError` would be raised if both of the above `if`/`elif` clauses were bypassed somehow in this routine.

Reply:
Good catch !

Reply:
Ok, Updated this issue, thanks